eng/common/TestResources/New-TestResources.ps1

```diff
@@ -115,6 +118,7 @@ $repositoryRoot = "$PSScriptRoot/../../.." | Resolve-Path
 $root = [System.IO.Path]::Combine($repositoryRoot, "sdk", $ServiceDirectory) | Resolve-Path
 $templateFileName = 'test-resources.json'
 $templateFiles = @()
+$environmentVariables = @{}
```

Comment:
`New-TestResources.ps1` now returns a list of environment variables that it created. We can use this information in calling scripts to set environment variables in the user's powershell environment or DevOps pipeline.